src/v/storage/segment_index.cc

```diff
@@ -196,7 +224,7 @@ ss::future<> segment_index::flush() {
 }
 ss::future<> segment_index::close() {
     co_await flush();
-    co_await _out.close();
+    co_await (co_await open()).close();
```

Comment:
What does this line do?

Reply:
I think this was probably some overly mechanical search and replace gone awry.  Removed.